pkg/querier/stats/stats_test.go

```diff
@@ -115,6 +115,23 @@ func TestStats_AddSplitQueries(t *testing.T) {
 	})
 }
 
+func TestStats_QueueTime(t *testing.T) {
+	t.Run("add and load queue time", func(t *testing.T) {
+		stats, _ := ContextWithEmptyStats(context.Background())
+		stats.AddQueueTime(time.Second)
+		stats.AddQueueTime(time.Second)
+
+		assert.Equal(t, 2*time.Second, stats.LoadQueueTime())
+	})
+
+	t.Run("add and load wall time nil receiver", func(t *testing.T) {
```

Comment:
wall -> queue

Reply:
Fixed in https://github.com/grafana/mimir/pull/6537/commits/0c1786c77965bfd62692ae09efce2beb13f54422.